editor/editor_node.cpp

```diff
@@ -735,6 +740,23 @@ void EditorNode::_notification(int p_what) {
 
 			// Save the project after opening to mark it as last modified, except in headless mode.
 			if (DisplayServer::get_singleton()->window_can_draw()) {
+				// Try to determine if this project's Godot version was less than 4.4 - if
+				// so, we'll ask the user if they want to upgrade the project.
+				PackedStringArray features = ProjectSettings::get_singleton()->get_setting("application/config/features");
+				if (features.size() >= 1) {
```

Comment:
This better communicates the intent:

```suggestion
				if (!features.is_empty()) {
```